### PR TITLE
[Refactor] Clearing up Unused Imports and Improving Performances and using lowerCamelCase syntax where required

### LINKED ISSUE #267: use lowerCamelCase identifier names
### 🐛 Describe the bug

Rename identifier according to lowerCamelCase convenction as prescribed by dart's linte rules.

image

### PR DESCRIPTION
**Pull request by Naman30903**
## Description

Removed unused imports , added const where required.

Fixes # (issue)

No issue fixes just improving Codebase

- [x] Refactor (does not change functionality, e.g. code style improvements, linting)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have checked my code and corrected any misspellings

## Maintainer Checklist

- [x] closes #267 and #280 and #248  
- [x] Tag the PR with the appropriate labels

### CODE REVIEW

**Review by Naman30903 — COMMENTED** [review]
Please Review the code and suggest changes or point out the problems

**Comment by Naman30903** [comment]
@chandansgowda Please can you review my PR

**Comment by vrundraval24** [comment]
Hey, @Naman30903 Please resolve the merge conflicts.

File: lib/utils/utils.dart [hunk 1]
```diff
@@ -32,17 +32,18 @@ class AppUtils {
       ),
       radius: UiSizes.size_20,
       titlePadding: EdgeInsets.only(top: UiSizes.size_25),
-      contentPadding: EdgeInsets.symmetric(horizontal: UiSizes.size_20, vertical: UiSizes.size_20),
+      contentPadding: EdgeInsets.symmetric(
+          horizontal: UiSizes.size_20, vertical: UiSizes.size_20),
```
Comment (vrundraval24) [inline lib/utils/utils.dart:36]:
```suggestion
          horizontal: UiSizes.size_20, 
          vertical: UiSizes.size_20,
      ),
```

File: lib/utils/utils.dart [hunk 2]
```diff
@@ -32,17 +32,18 @@ class AppUtils {
...
       actions: [
         ElevatedButton(
           onPressed: onFirstBtnPressed,
           style: ElevatedButton.styleFrom(
-            padding: EdgeInsets.symmetric(horizontal: 20, vertical: 10),
+            padding: const EdgeInsets.symmetric(horizontal: 20, vertical: 10),
```
Comment (vrundraval24) [inline lib/utils/utils.dart:41]:
```suggestion
            padding: const EdgeInsets.symmetric(
            horizontal: 20, 
            vertical: 10,
        ),
```

**Review by vrundraval24 — CHANGES_REQUESTED** [review]
Please don't remove the **Last Comma** as it helps Dart to arrange things in better way.

**Comment by Aarush-Acharya** [comment]
Hey @Naman-30903 please do the suggested changes and resolve the conflicts to bring it into the clean state so that we can merge

**Pull request closed without merging**